README.md

```diff
@@ -88,13 +88,17 @@ If a parameter is passed, available options:
 - `x.y.z` [Semver 2.0.0](https://semver.org/) string specifying the exact version to install
 - `latest` is a syntax to install latest version
 - `latest:<regex>` is a syntax to install latest version matching regex (used by grep -e)
-- `min-required` is a syntax to recursively scan your Terraform files to detect which version is minimally required. See [required_version](https://www.terraform.io/docs/configuration/terraform.html) docs. Also [see min-required](#min-required) section below.
+- `latest-allowed` is a syntax to recursively scan your Terraform files to detect which version is maximally allowed.
```

Comment:
It's not recursive, and certainly shouldn't be as terraform does not recurse when parsing code.